### PR TITLE
fix: only prepend to js, and add prependTo option

### PR DESCRIPTION
**Pull request by brandonocasey**

### CODE REVIEW

**Comment by gkatsev** [comment]
This doesn't seem right to me. We're moving from prepending the player code to all files to only prepending it to javascript files. Doesn't this mean that we're still including the player code in the bundle multiple times? That's not what we want.
Could we look for the player-loader file and prepend to that? Having a prependTo option sounds fine.

**Comment by brandonocasey** [comment]
It is only appended to output files, previously an output of css was having the player appended which was the issue I think. I could also see a scenario where you have one bundle that needs the player and another that does not, both being outputs, thats why I added an option to only prepend to what is specified.

**Comment by gkatsev** [comment]
Ah, it's only per output file. In that case, maybe we should only append it to the first javascript file? With dynamic imports, it's very common to get multiple javascript files from webpack. And then keep the prependTo option for when you know which bundle file it should be attached to specifically.

**Comment by brandonocasey** [comment]
Yeah lets change it to only the first output js file

File: src/index.js [hunk 1]
```diff
@@ -11,8 +12,15 @@ const defaults = {
 class PlayerLoaderPlugin {
 
   constructor(options) {
-    const settings = Object.assign({}, defaults, options);
-    const {accountId, embedId, playerId} = settings;
+    this.settings_ = Object.assign({}, defaults, options);
+    const {accountId, embedId, playerId} = this.settings_;
+
+    /* if prependTo was passed in, and was a string make it an array */
```
Comment (misteroneill) [inline src/index.js]: `/*`?

**Review by misteroneill — APPROVED** [review]

**Pull request merged**